x/http_secure_redirect.go

```diff
@@ -75,9 +84,9 @@ func SecureRedirectTo(r *http.Request, defaultReturnTo *url.URL, opts ...SecureR
 		}
 	}
 
-	if len(source.Query().Get("return_to")) == 0 {
+	if len(source.Query().Get(o.returnToKey)) == 0 {
 		return o.defaultReturnTo, nil
-	} else if returnTo, err = url.ParseRequestURI(source.Query().Get("return_to")); err != nil {
+	} else if returnTo, err = url.ParseRequestURI(source.Query().Get(o.returnToKey)); err != nil {
```

Comment:
What happens if we forgot to set this? I think a `stringsx.Coalesce` would be in helpful here!

Reply:
Hmm, well currently if the `SecureRedirectUseReturnToKey` option isn't set, `o.returnToKey` will default to `defaultReturnToKey` which is "return_to".

Just added a Coalesce so that if `SecureRedirectUseReturnToKey` is supplied an empty string, we'll fallback to the `defaultReturnToKey`